etc/Dictionaries/DFLError_en_GB.dic

```diff
@@ -24,9 +24,10 @@ UnknownMultiAssoc             =     couldn't find the multiAssociation %1% in as
 UnknownParamSet               =     couldn't find the parameter set %1% in setting file
 
 //------------------ Algo ---------------------------
+SeveralGeneratorinSVCRegulatingNode = node %1% is regulated by several generators whereas at least one of them (%2%) is in a SVC area
```

Comment:
I'd replace SVC by secondary voltage control when possible, it is more clear and there is no confusion with StaticVarCompensator